site/en/docs/extensions/mv3/end-to-end-testing/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: "layouts/doc-post.njk"
+title: "End-to-end testing Chrome Extensions"
+seoTitle: "End-to-end testing Chrome Extensions"
+date: 2023-10-09
+description: How to write end-to-end tests for extensions.
+---
+
+End-to-end testing involves an extension package being built and loaded into a browser. A testing
+tool communicates with the browser to automate interactions and test the same flows that a user
+would go through. A library that supports end-to-end testing will generally provide ways of
+controlling the browser, simulating user input and observing the current state of any open pages.
+
+See [Testing Chrome Extensions with Puppeteer][tutorial] for a tutorial.
+
+## Using browser testing libraries
```

Comment:
Does the guidance column look not quite right centered? 🤔

<img width="744" alt="Screenshot 2023-10-11 at 09 00 23" src="https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/e409bbc2-5172-4edc-a0c5-7fb76d2b62d0">

Reply:
I agree, but I think this is just how tables work? I did try not using a table but I think this is much better than another level of headings.

Reply:
To [left-align the column](https://www.markdownguide.org/extended-syntax/#alignment) you can replace the leftmost dash with a colon